Patches/Core/ThingDefs_Misc/Apparel_Various.xml

```diff
@@ -319,6 +319,13 @@
 		</value>
 	</Operation>
 
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_FlakPants"]/statBases/Mass</xpath>
+		<value>
+			<Mass>2</Mass>
```

Comment:
Why change this? Flak vest is crude, heavy armor composed of steel plates. Not exactly light stuff--a Vietnam-era flak jacket weight 11 kg.